### DISCUSSION TITLE
Support Blast & BlastSepolia

### DISCUSSION THREAD

**Discussion by niratodev** [alchemyplatform/aa-sdk#928] [General]
### Environment

- Browser version: All versions
- AA SDK version: 3.18.2
- Package: Alchemy Corre

### Describe the problem

In `core/src/chains/index.ts` Blast and BlastSepolia are not exported. When you try and instantiate `createLightAccountAlchemyClient` it throws an error because it is validated again the chain list with `getChain`.

![photo_2024-08-01_16-16-47](https://github.com/user-attachments/assets/38b7189d-9386-40c4-b009-ff1176f9e919)

**Comment by avarobinson** [comment]
Hey @niratodev! Unfortunately, we don't yet have AA support on Blast (i.e. Bundler and Gas Manager infrastructure). We are considering this for our roadmap and I'll pass your request back to the team.
